packages/suite/src/actions/settings/deviceSettingsActions.ts

```diff
@@ -77,7 +77,11 @@ export const changeWipeCode =
         });
         if (result.success) {
             if (!skipSuccessToast) {
```

Comment:
`skipSuccessToast` param is not used, please remove it. We always want to show the toast.